atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -284,12 +286,15 @@ private void saveSweepResults(final SweepProgressRowResult progress,
                                   final SweepResults results) {
         final long cellsDeleted = fromNullable(progress.getCellsDeleted()) + results.getCellsDeleted();
         final long cellsExamined = fromNullable(progress.getCellsExamined()) + results.getCellsExamined();
+        final long minimumSweptTimestamp = progress.getMinimumSweptTimestamp() == null ?
+                results.getSweptTimestamp() :
+                Math.min(progress.getMinimumSweptTimestamp(), results.getSweptTimestamp());
```

Comment:
The problem is if something was partially swept and you move to a sweep schema with the swept timestamp. Now you'll think you swept the previous results at a later timestamp than you actually did. Like the other thing, probably easiest to go through in person, but progress should actually never increment the min swept timestamp over the one in the first sweep run on that table. The one stored in priority is the one that should increase after full sweep runs.